sources/imdsclient/README.md

```diff
@@ -3,9 +3,19 @@
 Current version: 0.1.0
 
 The imdsclient library provides high-level methods to interact with the AWS Instance Metadata Service.
-The high-level methods provided are [`fetch_dynamic`], [`fetch_metadata`], and [`fetch_userdata`].
 
-For more control, and to query IMDS without high-level wrappers, there is also a [`fetch_imds`] method.
+The high-level methods provided are:
+- `fetch_cidr_blocks_for_mac`
+- `fetch_instance_type`
+- `fetch_local_ipv4_address`
+- `fetch_mac_addresses`
+- `fetch_public_ssh_keys`
+- `fetch_region`
+- `fetch_userdata`
+
+For custom IMDS end targets, you can use `fetch_bytes` and `fetch_string`.
+
+To query IMDS without high-level wrappers, there is also a `fetch_imds` method.
```

Comment:
Nit: Just from reading this, I'm not sure when I would use `fetch_bytes/string` vs. `fetch_imds`.  Can we clarify a bit?